### PR TITLE
style/controls design

### LINKED ISSUE #122: Design the control window

### PR DESCRIPTION
**Pull request by Trebua**
close #122 

**Features**
* A lot of refactoring (logic and new component)
* CSS-variables in index.css for simplifying adjustment of the design
* New responsive design. The colors are far from perfect, but better than before I hope.
* Some fancy effects like blinking and scaling when a value changes (in sent to rov-values) and scaling the current mode, as well as decreasing opacity of non-current mode. Also a little notice at the top right corner of modes if they are available or not
* Logic for switching between modes properly
* Deleted the ForceValues-component, but used its logic in Values

**Maybe look at after IMC is implemented:**
* Which global states should be used to read and write values
* Keeping the manual mode-values when toggling it off, so the previous values are set when the manual mode is toggled on again

image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -8,3 +8,29 @@ body,
...
+  --negative-dark: #8e0000;
+  --negative-light: #ff5f52;
+
+  --shadow-primary: 0 0 2px 1px black;
+  --shadow-secondary: 0 0 1.5px 0.5px black;
+  --shadow-big: 0 0 4px 1px black;
+
+  --big-radius: 5px;
+  --small-radius: 2px;
+}
```
Comment (eirikdahlen) [inline src/index.css]: I think the colors might be a bit dark? I think it changes a lot if the text is white, actually. See image for proposal. The text for `North` `East` `Surge` `Sway` etc could be black tho.
In general; if we have dark colors we should have bright text, if we have bright colors we should have dark text.
image

Comment (Trebua) [reply src/index.css]: Black on the dark blue passes the contrast checker, but I agree that white probably looks better. I will change it and then we can merge and iterate later 👍 

image

Comment (Trebua) [reply src/index.css]: image

image

Comment (Trebua) [reply src/index.css]: I think maybe black looks better 🤔

**Pull request merged**